packages/hub/plugin-loader.js

```diff
@@ -276,6 +279,10 @@ class ConfiguredPlugins {
         copied.attributes.enabled = true;
       }
       this._plugins[copied.id] = copied;
+
+      if (copied.attributes['is-root-plugin']) {
+        this.rootPlugin = copied;
```

Comment:
It is playing into the decision for the the topmost router, but it's not hardcoding it. The order of precedence is:
1. Use the router associated with the application card defined by the `plugin-configs/@cardstack/hub`
2. If there is no router associated with the application card defined by the `plugin-configs/@cardstack/hub`, then use the router associated with the cardstack application (the top most plugin)
3. If there is no router associated with the cardstack application, then use the "default" router, which is the routing feature in @cardstack/routing

Reply:
(and I have tests for these 3 different scenarios in the hub/node-tests/routers-test.js)

Reply:
I opted for rule #2 above, as it makes it easy to get up and running for a cardstack app by just having a `router.js` in your cardstack application as opposed to having to create a new module just for the purposes of having a router.

Reply:
I started creating a `cards/application-router` module in the deck project, and then stopped because that felt like too much boilerplate when all i wanted was a `router.js` for my cardstack application.